docs/journalist/sources.rst

```diff
@@ -39,24 +39,26 @@ Sending a reply
 ---------------
 
 Compose a reply to the selected source in the text box at the bottom of the
-conversation view. You may send it, or leave it in the box without sending to
-save it as a draft.
+conversation view. Click the paper airplane icon or press "Ctrl+Enter" to send
+a reply. Any replies you did not send will be discarded when you exit the
+client.
 
 Sources without reply keys
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
+You may sometimes see the text "Awaiting encryption key from server" in a grayed
+out reply box for a source. This typically happens in the following situations:
 
-If the reply box for a source is grayed out, this means that you will be unable
-to reply to the source until they have logged in again. This is because system
-resource limitations prevented an encryption key from being generated for the
-source the first time they logged in, possibly due to a large number of sources
-trying to reach your server.
+- A source has just contacted your SecureDrop moments ago, and the server has
+  not created an encryption key yet. If you are logged in, the key will be
+  automatically fetched as soon as it becomes available, usually within a few
+  seconds. At that point, the reply box will become available.
 
-This limitation is scheduled to be fixed in a future version of SecureDrop.
-Until then, you can use the Journalist Interface to `flag sources for reply`_.
-See your SecureDrop administrator for assistance provisioning a *Journalist
-Workstation* USB if you do not have one already.
+- Your SecureDrop server is experiencing a surge of traffic, and the generation
+  of encryption keys has been temporarily disabled. You have to manually
+  `flag the source for reply`_, and the source has to log in again before you
+  can respond. This cannot currently be done from the SecureDrop Client.
 
-.. _`flag sources for reply`: https://docs.securedrop.org/en/release-1.2.0/journalist.html#flag-for-reply
+.. _`flag the source for reply`: https://docs.securedrop.org/en/release-1.2.1/journalist.html#flag-for-reply
```

Comment:
Good catch! `latest` references the `develop` branch which I try to avoid if possible, because it's documentation for an unreleased version (it may reference functionality or behavior that are different than what the user sees in their production install, and this is a good case in point -- the "flag for reply" feature will be removed in `develop` before that removal lands in users' installs).

I tend to prefer `stable` which reflects the state of the latest release _tag_ (not branch) - but nothing after it, for better and for worse. Hopefully we'll be able to have a better option soon by having docs build from `master` and always referencing that; see discussion in https://github.com/freedomofpress/securedrop/issues/5081